saleor/app/events.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, Dict, Optional, Union
+
+from ..account.models import User
+from ..app.models import App
+from ..app.types import AppEventType
+from .models import AppEvent
+
+
+def _requestor(requestor: Optional[Union[App, User]]) -> Dict:
+    kwargs: Dict[str, Any] = {}
+    if requestor:
+        if isinstance(requestor, App):
+            kwargs["requestor_app"] = requestor
+        elif isinstance(requestor, User):
+            kwargs["requestor_user"] = requestor
+    return kwargs
+
+
+def app_event_installed(app: App, *, requestor: Optional[Union[App, User]]):
+    return AppEvent.objects.create(
+        app=app, type=AppEventType.INSTALLED, **_requestor(requestor)
+    )
+
+
+def app_event_activated(app: App, *, requestor: Optional[Union[App, User]]):
+    return AppEvent.objects.create(
+        app=app, type=AppEventType.ACTIVATED, **_requestor(requestor)
+    )
+
+
+def app_event_deactivated(app: App, *, requestor: Optional[Union[App, User]]):
+    return AppEvent.objects.create(
+        app=app, type=AppEventType.DEACTIVATED, **_requestor(requestor)
+    )
```

Comment:
Don't know if this is something for MVP but: Maybe would be worth adding the events for webhook to `AppEvent`? Like `webhook_created`, `updated`, `deleted`? This would be usefull as it could show what happens with the webhooks assigned to the specific app. WDYT?

Reply:
@korycins we will think about it in post-mvp. We don't want to accidentally end-up having monitoring/observability implementation here. Initially I was thinking about something user-facing. 

For app adding, upading and deleting events will be "daily job". For example, app should not register itself with webhook, because it will not work in many cases (config needed). So each time user changes configuration,webhook likely changes (disable/enable, create/delete).